global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.4"
+    "version": "2.2.104"
```

Comment:
Doesn't affect the library itself. Required only for development, changed purely because of the fact that I wasnt able to install older versions of SDKs on my Linux machine